<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_DMS_float</class>
 <widget class="QDialog" name="dlg_DMS_float">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert</string>
  </property>
  <widget class="QLineEdit" name="lineEditOutput_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TitleText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextInput_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input_2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_convert">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextInput_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input_1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInput_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInput_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TextOutput_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextOutput_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_exit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="DMSLabel1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>°</string>
   </property>
  </widget>
  <widget class="QLabel" name="DMSLabel1_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>°</string>
   </property>
  </widget>
  <widget class="QLabel" name="DMSLabel1_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>'</string>
   </property>
  </widget>
  <widget class="QLabel" name="DMSLabel1_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>'</string>
   </property>
  </widget>
  <widget class="QLabel" name="DMSLabel1_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="DMSLabel1_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditInput_1</tabstop>
  <tabstop>lineEditInput_2</tabstop>
  <tabstop>pb_convert</tabstop>
  <tabstop>pb_exit</tabstop>
  <tabstop>lineEditOutput_1</tabstop>
  <tabstop>lineEditOutput_2</tabstop>
  <tabstop>lineEditOutput_3</tabstop>
  <tabstop>lineEditOutput_4</tabstop>
  <tabstop>lineEditOutput_5</tabstop>
  <tabstop>lineEditOutput_6</tabstop>
  <tabstop>lineEditOutput_7</tabstop>
  <tabstop>lineEditOutput_8</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
